tools/webpack/packages.js

```diff
@@ -156,5 +157,9 @@ module.exports = {
 				.concat( bundledPackagesPhpConfig )
 				.concat( vendorsCopyConfig ),
 		} ),
+		new MomentTimezoneDataPlugin( {
+			startYear: 2000,
+			endYear: 2030,
+		} ),
```

Comment:
This will require us to maintain the years every now and then. Does it perhaps make sense to make these dynamic?

```
const currentYear = new Date().getFullYear();
```

and then something like:

```suggestion
			startYear: currentYear - 20,
			endYear: currentYear + 10,
```

Reply:
I'd rather keep it dumb instead of dynamic. We need to start at least in 2003, the year of WordPress first release, and in 2030, we surely won't use moment.js any more 😉

Reply:
What you're saying is true, but how does it relate to the timezone config? Yes, WordPress started in 2003, but you can have a post written earlier (in 1997 for example) and that's fine. You can also schedule a post for 2040, that's perfectly fine. Perhaps we can keep the end-year dynamic at least? Otherwise, we may easily forget to update it as we get closer to it, and as much as we should remove `moment.js` by then, WordPress is known to maintain backwards compatibility forever. This might mean that the `@wordpress/date` package will remain forever, albeit unused in Gutenberg, and that would mean that this config will also remain for the time being.

Reply:
I was also curious about other CPTs which may not adhere to when they were posted. For example, someone could create a historical timeline using CPTs where each event is a post and has a historical date. (Though does that even matter in the context of this webpack plugin?)

Reply:
Yes, if you're creating a history site and want a post to have an exact time of a historical event, e.g., November 22, 1963, at 12:30 p.m. CST, you might run into trouble if we ship a TZ range that doesn't include year 1963. But I'm not sure what exactly will happen. For this particular date probably nothing, because it's safely outside the DST range. But if you choose a date which would be DST in 1963 but not today, or vice versa, the editor might show you a time that's off by one hour.